yaml2sbml/YamlModel.py

```diff
@@ -109,14 +109,18 @@ def write_to_sbml(self,
             raise ValueError('sbml_dir should contain path to the sbml '
                              'and hence end with .xml or .sbml')
 
+        # model name = sbml name without file extension
+        model_name = os.path.splitext(sbml_dir)[0][:-4]
```

Comment:
why -4? is this always 4 letters?

Reply:
The following may be sufficient
```python
from pathlib import Path
# The name of the model is the SBML filename without its extension.
model_name = Path(sbml_dir).stem
```

Reply:
thank you @dilpath (looking at your name, you of course know path lib :D (sorry, bad pun...))